jetty-core/jetty-websocket/jetty-websocket-jetty-api/src/main/java/org/eclipse/jetty/websocket/api/Callback.java

```diff
@@ -14,24 +14,26 @@
 package org.eclipse.jetty.websocket.api;
 
 import java.util.concurrent.CompletableFuture;
+import java.util.function.BiConsumer;
 import java.util.function.Consumer;
 
 /**
- * Callback for Write events.
- * <p>
- * NOTE: We don't expose org.eclipse.jetty.util.Callback here as that would complicate matters with the WebAppContext's classloader isolation.
```

Comment:
niggle: this would be good to say in a comment rather than javadoc, so we remember why it has it's own Callback interface.